### ISSUE TITLE
False positives of `Style/IfUnlessModifier`

### ISSUE THREAD

**Issue by AlexWayfer** [rubocop-hq/rubocop#8041]
Hello.

Looks similar to #7885. 

I'm trying to update RuboCop to version 0.84.0 for Formalism: https://github.com/AlexWayfer/formalism/pull/50

I'm using tabs in this project.

And there is a condition which does not fit in one line.

--------

## Expected behavior

No offenses.

## Actual behavior

Offense from `Style/IfUnlessModifier`.

## Steps to reproduce the problem

```ruby
						unless params_method_defined
							define_method(params_method_name) { @params[name] }
						end
```

![image](https://user-images.githubusercontent.com/6510020/82921017-a7ea4580-9f80-11ea-9583-e1559132ea33.png)

![image](https://user-images.githubusercontent.com/6510020/82921128-b3d60780-9f80-11ea-8420-8b813a728447.png)

## RuboCop version

```
0.84.0 (using Parser 2.7.1.3, rubocop-ast 0.0.3, running on ruby 2.7.1 x86_64-linux)
```

**Comment by AlexWayfer** [comment]
Other examples:

* https://github.com/AlexWayfer/sequel-enum_values/pull/27
* https://github.com/AlexWayfer/flame/pull/137

**Comment by AlexWayfer** [comment]
Sorry, it affects not only tabs indentation, so it global false positives for `IfUnlessModifier` cop. Example: https://github.com/filewatcher/filewatcher/pull/122

**Comment by jonas054** [comment]
Isn't this related to the change of default value for `Layout/LineLength:Max` to 120 in 0.84.0?

**Comment by AlexWayfer** [comment]
> Isn't this related to the change of default value for `Layout/LineLength:Max` to 120 in 0.84.0?

Yes, it's related. Thank you, I didn't notice that, there were no warnings in console…

Sorry.
